.github/workflows/build.yml

```diff
@@ -135,11 +135,11 @@ jobs:
             type=semver,pattern={{version}}
             type=semver,pattern={{major}}
             type=semver,pattern={{major}}.{{minor}}
-            type=raw,value=1.10.2-SNAPSHOT,enable=${{ github.event.inputs.deploy_docker == 'true' || github.ref == format('refs/heads/{0}', 'main') }}
+            type=raw,value=1.10.6-SNAPSHOT,enable=${{ github.event.inputs.deploy_docker == 'true' || github.ref == format('refs/heads/{0}', 'main') }}
```

Comment:
Is it necessary to hard code a version number here? This implies manual change with each new version and the build pipeline should not be changed with each new release/version?! Or am I missing something here? Same in line 176

Reply:
You are completely right, this is redundant, but the reason is lacking knowledge from my side.

Since we are running without tags on 'main', there was for me no easy way to find out the actual target version which is also apparent in the pom.xml's 

Since we have these versions currently distributed over quite some files (pom.xml, *.md, even Chart.yaml) I have to do a sed command anyway ...

Maybe I should add that command for version upgrading to the root folder?

Reply:
An thanks for caring!

Reply:
I rechecked https://github.com/docker/metadata-action ... it takes its semver version information  only from git tags, means that when running on master, we would not get any version information at all.
Since we educated our users to first integrate with the -SNAPSHOT images and then to e2e test with the release versions by removing the -SNAPSHOT we can not easily throw that scheme away.